src/main/java/com/google/jenkins/plugins/computeengine/client/ComputeClient.java

```diff
@@ -390,6 +395,34 @@ public Instance getInstance(String projectId, String zone, String instanceId) th
         }
         return instances;
     }
+    
+    public InstanceTemplate getTemplate(String projectId, String templateName) throws IOException {
+        return compute
+                .instanceTemplates()
+                .get(projectId, templateName)
+                .execute();
+    }
+    
+    public List<InstanceTemplate> getTemplates(String projectId) throws IOException {
+        List<InstanceTemplate> instanceTemplates = compute
+                .instanceTemplates()
+                .list(projectId)
+                .execute()
+                .getItems();
+        if (instanceTemplates == null) {
+            instanceTemplates = new ArrayList<>();
+        }
+
+        // Sort by name
+        Collections.sort(instanceTemplates, new Comparator<InstanceTemplate>() {
```

Comment:
This could be made more concise with a lambda, suggest: instanceTemplates.sort((InstanceTemplate o1, InstanceTemplate o2) -> o1.getName().compareTo(o2.getName())))

Reply:
Idea gave me even nicer hint..